### ISSUE TITLE
Messages about missing version constraints are wrong

### ISSUE THREAD

**Issue by izaera** [liferay/liferay-amd-loader#166]
In the following screenshot:

<img width="933" alt="imagen" src="https://user-images.githubusercontent.com/3273630/36249924-9950be42-123c-11e8-9bfa-730eccd32171.png">

Where it says:

```
Missing version constraints for rxjs in package.json of @angular/forms@4.2.6/observable/forkJoin
```

It should say:

```
Missing version constraints for rxjs/observable/forkJoin in package.json of @angular/forms@4.2.6
```

And so on...

**Comment by izaera** [comment]
This is a portal error (the message comes from the server).

**Comment by izaera** [comment]
See https://issues.liferay.com/browse/LPS-78129

**Comment by izaera** [comment]
Reopening because the error is here (see https://issues.liferay.com/browse/LPS-78129?focusedCommentId=1253908&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1253908)

**Comment by landoscotto** [comment]
I have same error:
`Liferay AMD Loader: Unhandled failure: Error: The following problems where detected while resolving modules:
Missing version constraints for @angular/core in package.json of primeng@4.1.3
Missing version constraints for @angular/common in package.json of primeng@4.1.3
    at Loader._getResolutionError (combo?browserId=other&minifierType=js&languageId=en_US&b=7100&t=1538724555918&/o/frontend-js-web/jquery/jquery.js&/o/frontend-js-web/clay/popper.js&/o/frontend-js-web/clay/bootstrap.js&/o/frontend-js-web/loader/config.js&/o/frontend-js-web/loader/loader.js&/o/frontend-js-web/lodash/lodash.js&/o/frontend-js-web/aui/aui/aui.js&/o/frontend-js-web/aui/aui-base-html5-shiv/aui-base-html5-shiv.js&/o/frontend-js-web/liferay/browser_selectors.js&/o/frontend-js-web/liferay/modules.js&/o/frontend-js-web/liferay/aui_sandbox.js&/o/frontend-js-web/clay/collapsible-search.js&/o/frontend-js-web/clay/side-navigation.js&/o/frontend-js-web/jquery/fm.js&/o/frontend-js-web/jquery/form.js&/o/frontend-js-web/lodash/util.js&/o/frontend-js-web/misc/svg4everybody.js&/o/frontend-js-web/aui/arraylist-add/arraylist-add.js&/o/frontend-js-web/aui/arraylist-filter/arraylist-filter.js&/o/frontend-js-web/aui/arraylist/arraylist.js&/o/frontend-js-web/aui/array-extras/array-extras.js&/o/frontend-js-web/aui/array-invoke/array-invoke.js&/o/frontend-js-web/aui/attribute-base/attribute-base.js&/o/frontend-js-web/aui/attribute-complex/attribute-complex.js&/o/frontend-js-web/aui/attribute-core/attribute-core.js&/o/frontend-js-web/aui/attribute-observable/attribute-observable.js&/o/frontend-js-web/aui/attribute-extras/attribute-extras.js&/o/frontend-js-web/aui/base-base/base-base.js&/o/frontend-js-web/aui/base-pluginhost/base-pluginhost.js&/o/frontend-js-web/aui/classnamemanager/classnamemanager.js&/o/frontend-js-web/aui/datatype-xml-format/datatype-xml-format.js&/o/frontend-js-web/aui/datatype-xml-parse/datatype-xml-parse.js&/o/frontend-js-web/aui/dom-base/dom-base.js&/o/frontend-js-web/aui/dom-core/dom-core.js&/o/frontend-js-web/aui/dom-screen/dom-screen.js&/o/frontend-js-web/aui/dom-style/dom-style.js&/o/frontend-js-web/aui/event-base/event-base.js&/o/frontend-js-web/aui/event-custom-base/event-custom-base.js&/o/frontend-js-web/aui/event-custom-complex/event-custom-complex.js&/o/frontend-js-web/aui/event-delegate/event-delegate.js:17307)
    at combo?browserId=other&minifierType=js&languageId=en_US&b=7100&t=1538724555918&/o/frontend-js-web/jquery/jquery.js&/o/frontend-js-web/clay/popper.js&/o/frontend-js-web/clay/bootstrap.js&/o/frontend-js-web/loader/config.js&/o/frontend-js-web/loader/loader.js&/o/frontend-js-web/lodash/lodash.js&/o/frontend-js-web/aui/aui/aui.js&/o/frontend-js-web/aui/aui-base-html5-shiv/aui-base-html5-shiv.js&/o/frontend-js-web/liferay/browser_selectors.js&/o/frontend-js-web/liferay/modules.js&/o/frontend-js-web/liferay/aui_sandbox.js&/o/frontend-js-web/clay/collapsible-search.js&/o/frontend-js-web/clay/side-navigation.js&/o/frontend-js-web/jquery/fm.js&/o/frontend-js-web/jquery/form.js&/o/frontend-js-web/lodash/util.js&/o/frontend-js-web/misc/svg4everybody.js&/o/frontend-js-web/aui/arraylist-add/arraylist-add.js&/o/frontend-js-web/aui/arraylist-filter/arraylist-filter.js&/o/frontend-js-web/aui/arraylist/arraylist.js&/o/frontend-js-web/aui/array-extras/array-extras.js&/o/frontend-js-web/aui/array-invoke/array-invoke.js&/o/frontend-js-web/aui/attribute-base/attribute-base.js&/o/frontend-js-web/aui/attribute-complex/attribute-complex.js&/o/frontend-js-web/aui/attribute-core/attribute-core.js&/o/frontend-js-web/aui/attribute-observable/attribute-observable.js&/o/frontend-js-web/aui/attribute-extras/attribute-extras.js&/o/frontend-js-web/aui/base-base/base-base.js&/o/frontend-js-web/aui/base-pluginhost/base-pluginhost.js&/o/frontend-js-web/aui/classnamemanager/classnamemanager.js&/o/frontend-js-web/aui/datatype-xml-format/datatype-xml-format.js&/o/frontend-js-web/aui/datatype-xml-parse/datatype-xml-parse.js&/o/frontend-js-web/aui/dom-base/dom-base.js&/o/frontend-js-web/aui/dom-core/dom-core.js&/o/frontend-js-web/aui/dom-screen/dom-screen.js&/o/frontend-js-web/aui/dom-style/dom-style.js&/o/frontend-js-web/aui/event-base/event-base.js&/o/frontend-js-web/aui/event-custom-base/event-custom-base.js&/o/frontend-js-web/aui/event-custom-complex/event-custom-complex.js&/o/frontend-js-web/aui/event-delegate/event-delegate.js:17207
    at ZoneDelegate.invoke (?browserId=other&minifierType=&languageId=en_US&b=7100&t=1538724555918&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-dom/src/all/dom.js&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-uri/src/Uri.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Geometry.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Position.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Align.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/all/position.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.soy.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.js&/o/js/resolved-module/frontend-taglib-clay@1.0.5/bridge/clay-tooltip/src/ClayTooltip.js&/o/js/resolved-module/reflect-metadata@0.1.12/Reflect.js&/o/js/resolved-module/zone.js@0.8.26/dist/zone-node.js&/o/js/resolved-module/documenti-web@1.0.0/js/angular-loader.js:2905)
    at Zone.run (?browserId=other&minifierType=&languageId=en_US&b=7100&t=1538724555918&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-dom/src/all/dom.js&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-uri/src/Uri.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Geometry.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Position.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Align.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/all/position.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.soy.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.js&/o/js/resolved-module/frontend-taglib-clay@1.0.5/bridge/clay-tooltip/src/ClayTooltip.js&/o/js/resolved-module/reflect-metadata@0.1.12/Reflect.js&/o/js/resolved-module/zone.js@0.8.26/dist/zone-node.js&/o/js/resolved-module/documenti-web@1.0.0/js/angular-loader.js:2681)
    at ?browserId=other&minifierType=&languageId=en_US&b=7100&t=1538724555918&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-dom/src/all/dom.js&/o/js/resolved-module/frontend-js-metal-web@2.0.3/bridge/metal-uri/src/Uri.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Geometry.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Position.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/Align.js&/o/js/resolved-module/frontend-js-metal-web$metal-position@2.1.1/lib/all/position.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.soy.js&/o/js/resolved-module/frontend-taglib-clay$clay-tooltip@2.1.12/lib/ClayTooltip.js&/o/js/resolved-module/frontend-taglib-clay@1.0.5/bridge/clay-tooltip/src/ClayTooltip.js&/o/js/resolved-module/reflect-metadata@0.1.12/Reflect.js&/o/js/resolved-module/zone.js@0.8.26/dist/zone-node.js&/o/js/resolved-module/documenti-web@1.0.0/js/angular-loader.js:3369`

How i can fix?

**Comment by izaera** [comment]
This is obsolete since server-side resolution (#185)
